modules/nodes-nodes-swap-memory.adoc

```diff
@@ -0,0 +1,50 @@
+:_content-type: PROCEDURE
+[id="nodes-nodes-swap-memory_{context}"]
+
+= Managing swap memory on nodes
+
+:FeatureName: Configuring swap memory usage on nodes
+include::modules/technology-preview.adoc[leveloffset=+1]
```

Comment:
Apparently, this file has been moved. 
`include::snippets/technology-preview.adoc[]`